### DISCUSSION TITLE
Google Domains at Vercel - GSuit Email

### DISCUSSION THREAD

**Discussion by MiguelMachado-dev** [vercel/vercel#5968] [Help]
I put my Google domains into Vercel, and created a GSuit Email with that domain. But every time I send an email a red question mark is displayed as if it is spam.

![image](https://user-images.githubusercontent.com/29252011/110795949-2c7b2b80-8256-11eb-9789-bf0294d7efee.png)

What I have to do?

**Comment by fu-sen** [comment]
I don't think it's a question directly related to Vercel, but here are some tips for solving it:
https://support.google.com/mail/answer/180707

For example: Are you setting up an SPF record? Currently it is mandatory.

If you use Google Workspace (It's the current name of GSuite) for a fee,  that support is very useful.
